Makefile

```diff
@@ -132,6 +132,14 @@ tools:
 run: build
 	@cargo osdk run $(CARGO_OSDK_ARGS)
 
+.PHONY: gdb_server
+gdb_server: build
+	@cd kernel && cargo osdk run $(CARGO_OSDK_ARGS) -G --vsc --gdb-stub :1234
```

Comment:
Seems inconsistent: in the code and the doc the options is `--gdb-server-addr` but `--gdb-stub` here.

Reply:
Sry. Forgot to adapt the cli interface...